products/sle/main.pm

```diff
@@ -1270,6 +1270,16 @@ elsif (get_var("FIPS_TS")) {
         load_applicationstests;
     }
 }
+elsif (check_var('SMT', 'external')) {
+    prepare_target();
+    loadtest "x11/smt_disconnect_prepare";
+    loadtest "x11/smt_disconnect_external";
+}
+elsif (check_var('SMT', 'internal')) {
```

Comment:
Why not to combine these two conditions?

Reply:
this is a multi machine case.need check the parameters for loading which case.

Reply:
I agree with @rwx788, @dehai please combine these 2 blocks to avoid duplication

Reply:
@mitiao @rwx788  have combined.